langworld_db_data/removers/listed_value_remover.py

```diff
@@ -15,7 +15,13 @@ def remove_listed_value(
         removed_value_information = self._remove_from_inventory_of_listed_values(
             id_of_value_to_remove
         )
-        self._remove_from_feature_profiles()
+        """
+        Contains the feature ID and English and Russian names of the removed value.
+        """
+        self._remove_from_feature_profiles(
+            id_of_value_to_remove=id_of_value_to_remove,
+            english_name_of_value_to_remove=removed_value_information["en"],
+        )
 
         if not removed_value_information:
             raise ListedValueRemoverError(
```

Comment:
fix codecov issue?

Reply:
`codecov` is still complaining about `raise ListedValueRemoverError` in line 30 not being covered by tests